### ISSUE TITLE
Key binding for F1 (Server Log) doesn't work anymore

### ISSUE THREAD

**Issue by frankhale** [frankhale/electron-with-express#25]
Originally when this started you could grab global key bindings to do various things. Sometime a while ago Keymaster stopped working. This isn't a problem with Keymaster. This is something that happened in Electron. Keybindings no longer work with respect to webviews so you can't open up the server log. It's a shame but I don't really feel like fixing it. If anyone does I'll accept a pull request.

**Comment by frankhale** [comment]
The project keeps getting stars but is not likely to get anymore updates so I'm going to mark it archived unless there are objections

**Comment by frankhale** [comment]
I'd love to make the key bind work again but maybe the easiest thing to do is to either enable the menu bar or create a custom menu that allows the Server Log to be shown. Maybe the answer is to move away from a web view but I haven't looked at what the options are recently since I've been not paying attention to Electron for quite some time.

**Comment by Aqueuse** [comment]
it work for me, I'm in Windows 10 and run the app with powershell. Just to give you a feedback, thank you for this good example, it help a lot to begin.

**Comment by frankhale** [comment]
Okay so as I'm just getting back into this after an extended break I forgot some things I already did previously. I'd like to make a correction to what I said. It seems I switched from `webview' to `iframe` for hosting the express content. If you do not focus the main part of the window (no clicking) then the key bind works. After you click the content the key bind will no longer get triggered because the `iframe` has exclusive control apparantly.  I am pretty sure Electron has a way to do global key binds inside the Electron main process so I'm going to attempt that.
